backside/places/domain/domain.go

```diff
@@ -0,0 +1,122 @@
+package domain
+
+import (
+	"sk8.town/backside/places/dto"
+)
+
+type Position struct {
+	Lat float64 `json:"lat"`
+	Lon float64 `json:"lon"`
+}
+
+type Poi struct {
+	Name        string `json:"name"`
+	Phone       string `json:"phone"`
+	CategorySet []struct {
+		ID int `json:"id"`
+	} `json:"categorySet"`
+	URL             string   `json:"url"`
+	Categories      []string `json:"categories"`
+	Classifications []struct {
+		Code  string `json:"code"`
+		Names []struct {
+			NameLocale string `json:"nameLocale"`
+			Name       string `json:"name"`
+		} `json:"names"`
+	} `json:"classifications"`
+}
+
+type Address struct {
+	Municipality                string `json:"municipality"`
+	CountrySecondarySubdivision string `json:"countrySecondarySubdivision"`
+	CountrySubdivision          string `json:"countrySubdivision"`
+	CountryCode                 string `json:"countryCode"`
+	Country                     string `json:"country"`
+	CountryCodeISO3             string `json:"countryCodeISO3"`
+	FreeformAddress             string `json:"freeformAddress"`
+}
```

Comment:
I split the big `Result` struct so it's easier to mock the response in test cases, see here: https://github.com/sk8mate/sk8/pull/34/files#diff-06905978e8291d623732774b00d9190e3ebacea0a22e7d0dcf5db0561ea7276aR39-R65

Reply:
Also, I think maybe we shouldn't declare fields we're not interested in?